test/cl/implementation.lisp

```diff
@@ -1,87 +1,87 @@
 (in-package :thrift.test-implementation)
```

Comment:
Can't we change this to `(in-package #:thrift.test.thrift-test-implementation)` and then avoid some of those long names below?

Reply:
On that note, I think implementation packages should import the `:cl` package for convenience, as they are meant for users to work with when implementing their servers.

Reply:
we want to have one implementation package which only accesses service-implementation package symbols. service-implementation package doesn't import anything and exports only service methods. Top-level IDL package on the other hand imports cl and thrift packages, so things like `float` and `byte` are imported. So implementation of all IDL services should be done in thrift.test-implementation package, and method names should be symbols taken from thrift.test.service-implementation packages.

Reply:
thanks to that thrift.test.service-implementation package is isolated, and methods like `list` may be defined. If we had imported cl package, such methods couldn't be implemented.

Reply:
way to avoid long names would be importing symbols from `thrift.test.test-implementation` to `thrift.test-implementation`, but that can't be done automatically because of potential method name conflicts in different services.

Reply:
Yeah, that makes sense. You convinced me. I guess the user can always define their own package and import the specific service package.